dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1140,6 +1141,19 @@ private void wesServiceInfo(WorkflowExecutionServiceApi clientWorkflowExecutionS
         }
     }
 
+    /**
+     * This will attempt to retrieve information regarding the WES server
+     * @param clientWorkflowExecutionServiceApi The API client
```

Comment:
Document all the params or document none, IMHO. :)

Reply:
Good catch. I'd initially not had those parameters, and didn't update the docs once they were added.

Fixed: https://github.com/dockstore/dockstore-cli/pull/112/commits/7118069cdb9bcacc7623c43a5c23e803b7090a21